public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -174,8 +175,17 @@ export class PanelModel {
     return this.options;
   }
 
+  replaceVariables = (value: string, extraVars?: ScopedVars, format?: string) => {
```

Comment:
can we make this a prototype function instead? Otherwise, this needs to be excluded from getSaveModel etc / it appears in snapshots etc

Reply:
Yes, absolutely